pkgs/build-support/node/fetch-npm-deps/default.nix

```diff
@@ -165,7 +165,7 @@
 
       dontInstall = true;
 
-      impureEnvVars = lib.fetchers.proxyImpureEnvVars;
+      impureEnvVars = lib.fetchers.proxyImpureEnvVars ++ [ "NIX_NPM_TOKENS" ];
```

Comment:
It might make sense to document the format here as well.

Reply:
Yeah, I agree that's a good idea. I'll do that when I'm next at my computer. Thank you!

I _should_ also add it to the tooling docs, so maybe I'll do that too

Reply:
Done! Apparently there still is no documentation on `fetchNpmDeps` in general so, uh, I didn't include an update to the tooling docs that apparently don't exist 😅